tooling/minimize/tests/ub/tree_borrows/mutable_ref_child_write_frozen.rs

```diff
@@ -0,0 +1,15 @@
+//@ compile-flags: --minimize-tree-borrows
+
+// Check that a foreign read makes an active mutable reference frozen.
+// After that, writing to this mutable reference becomes invalid.
+
+fn main() {
+    let x = &mut 31; // (x, Reserved)   
+    let xraw = x as *mut i32; // (x, Reserved) 
+    let y = unsafe { &mut *xraw }; // (x, Reserved) -> (y, Reserved)
+    *y = 42; // (x, Active) -> (y, Active)
+    assert!(unsafe { *xraw } == 42); // (x, Active) -> (y, Frozen)
+
+    assert!(*y == 42); // Child read to Frozen is ok.
```

Comment:
```suggestion
    assert!(*y == 42); // Child read from Frozen is ok.
```
Same everywhere else: it's "read from", not "read to"